packages/card/locales/nb/messages.po

```diff
@@ -0,0 +1,20 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2024-07-30 14:45+0200\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: nb\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
+
+#. Screenreader message to indicate that the card is clickable
+#. js-lingui-explicit-id
+#: packages/card/index.js:34
+msgid "card.button.text"
+msgstr ""
```

Comment:
This one we know from `packages/card/index.js` so it should be "Velg".

Reply:
Updated in this commit: [ab72db0](https://github.com/warp-ds/elements/pull/177/commits/ab72db064323acb00daf8cf2d23036e395021ee8)